test/inputs/tiny/nodes-profiles.csv

```diff
@@ -1,218 +1,218 @@
-3,rp3,k12,0.8
-3,rp3,k13,0.81
-3,rp3,k14,0.81
-3,rp3,k15,0.8
-3,rp3,k16,0.79
-3,rp3,k17,0.78
-3,rp3,k18,0.77
-3,rp3,k19,0.76
-3,rp3,k20,0.75
-3,rp3,k21,0.74
-3,rp3,k22,0.74
-3,rp3,k23,0.74
-3,rp3,k24,0.74
-4,rp1,k01,0.0
-4,rp1,k02,0.0
-4,rp1,k03,0.0
-4,rp1,k04,0.0
-4,rp1,k05,0.0
-4,rp1,k06,0.02
-4,rp1,k07,0.12
-4,rp1,k08,0.3
-4,rp1,k09,0.5
-4,rp1,k10,0.66
-4,rp1,k11,0.78
-4,rp1,k12,0.83
-4,rp1,k13,0.83
-4,rp1,k14,0.78
-4,rp1,k15,0.68
-4,rp1,k16,0.53
-4,rp1,k17,0.35
-4,rp1,k18,0.17
-4,rp1,k19,0.04
-4,rp1,k20,0.0
-4,rp1,k21,0.0
-4,rp1,k22,0.0
-4,rp1,k23,0.0
-4,rp1,k24,0.0
-4,rp2,k01,0.0
-4,rp2,k02,0.0
-4,rp2,k03,0.0
-4,rp2,k04,0.0
-4,rp2,k05,0.0
-4,rp2,k06,0.0
-4,rp2,k07,0.01
-4,rp2,k08,0.07
-4,rp2,k09,0.2
-4,rp2,k10,0.36
-4,rp2,k11,0.5
-4,rp2,k12,0.57
-4,rp2,k13,0.59
-4,rp2,k14,0.54
-4,rp2,k15,0.44
-4,rp2,k16,0.29
-4,rp2,k17,0.13
-4,rp2,k18,0.03
-4,rp2,k19,0.0
-4,rp2,k20,0.0
-4,rp2,k21,0.0
-4,rp2,k22,0.0
-4,rp2,k23,0.0
-4,rp2,k24,0.0
-4,rp3,k01,0.0
-4,rp3,k02,0.0
-4,rp3,k03,0.0
-4,rp3,k04,0.0
-4,rp3,k05,0.0
-4,rp3,k06,0.0
-4,rp3,k07,0.0
-4,rp3,k08,0.01
-4,rp3,k09,0.12
-4,rp3,k10,0.28
-4,rp3,k11,0.42
-4,rp3,k12,0.51
-4,rp3,k13,0.53
-4,rp3,k14,0.5
-4,rp3,k15,0.4
-4,rp3,k16,0.23
-4,rp3,k17,0.05
-4,rp3,k18,0.0
-4,rp3,k19,0.0
-4,rp3,k20,0.0
-4,rp3,k21,0.0
-4,rp3,k22,0.0
-4,rp3,k23,0.0
-4,rp3,k24,0.0
-6,rp1,k01,0.852017937
-6,rp1,k02,0.780269058
-6,rp1,k03,0.730044843
-6,rp1,k04,0.698654709
-6,rp1,k05,0.679820628
-6,rp1,k06,0.673542601
-6,rp1,k07,0.698654709
-6,rp1,k08,0.747982063
-6,rp1,k09,0.80896861
-6,rp1,k10,0.857399103
-6,rp1,k11,0.905829596
-6,rp1,k12,0.917488789
-6,rp1,k13,0.913004484
-6,rp1,k14,0.905829596
-6,rp1,k15,0.878923767
-6,rp1,k16,0.865470852
-6,rp1,k17,0.86367713
-6,rp1,k18,0.894170404
-6,rp1,k19,0.980269058
-6,rp1,k20,0.999103139
-6,rp1,k21,1.0
-6,rp1,k22,0.992825112
-6,rp1,k23,0.944394619
-6,rp1,k24,0.928251121
-6,rp2,k01,0.852017937
-6,rp2,k02,0.780269058
-6,rp2,k03,0.730044843
-6,rp2,k04,0.698654709
-6,rp2,k05,0.679820628
-6,rp2,k06,0.673542601
-6,rp2,k07,0.698654709
-6,rp2,k08,0.747982063
-6,rp2,k09,0.80896861
-6,rp2,k10,0.857399103
-6,rp2,k11,0.905829596
-6,rp2,k12,0.917488789
-6,rp2,k13,0.913004484
-6,rp2,k14,0.905829596
-6,rp2,k15,0.878923767
-6,rp2,k16,0.865470852
-6,rp2,k17,0.86367713
-6,rp2,k18,0.894170404
-6,rp2,k19,0.980269058
-6,rp2,k20,0.999103139
-6,rp2,k21,1.0
-6,rp2,k22,0.992825112
-6,rp2,k23,0.944394619
-6,rp2,k24,0.928251121
-6,rp3,k01,0.852017937
-6,rp3,k02,0.780269058
-6,rp3,k03,0.730044843
-6,rp3,k04,0.698654709
-6,rp3,k05,0.679820628
-6,rp3,k06,0.673542601
-6,rp3,k07,0.698654709
-6,rp3,k08,0.747982063
-6,rp3,k09,0.80896861
-6,rp3,k10,0.857399103
-6,rp3,k11,0.905829596
-6,rp3,k12,0.917488789
-6,rp3,k13,0.913004484
-6,rp3,k14,0.905829596
-6,rp3,k15,0.878923767
-6,rp3,k16,0.865470852
-6,rp3,k17,0.86367713
-6,rp3,k18,0.894170404
-6,rp3,k19,0.980269058
-6,rp3,k20,0.999103139
-6,rp3,k21,1.0
-6,rp3,k22,0.992825112
-6,rp3,k23,0.944394619
-6,rp3,k24,0.928251121
+id,rep_period,time_slice,value
+3,1,01,0.11
+3,1,02,0.11
```

Comment:
Did you mean to leave these preceded by zero?

Reply:
I don't think so, with all the multi-cursor fun, we missed the leading zeros. Does it matter though? I see CSV.jl parses it correctly.